calliope/test/common/test_model/model_demand_share.yaml

```diff
@@ -0,0 +1,76 @@
+model:
```

Comment:
i.e. we're now moving to creating a yaml file for each functionality? This could quickly become quite long, particularly if we were to retrospectively apply it to all existing functionality. Is that the plan?

Reply:
@brynpickering happy with the comments in https://github.com/calliope-project/calliope/pull/191/commits/7d7a05c8e3f1fc53a756a34dbf5139965d4fec92 ?

Reply:
Yup